includes/class-amp-rest-api.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * AMP REST API.
+ *
+ * @package AMP
+ * @since   2.0
+ */
+
+/**
+ * Class AMP_REST_API.
+ */
+class AMP_REST_API {
+
+	/**
+	 * Init.
+	 */
+	public static function init() {
+		add_action( 'rest_api_init', [ __CLASS__, 'rest_api_init' ] );
+	}
+
+	/**
+	 * Register other actions and filters to be used during the REST API initilization.
+	 *
+	 * @return void
+	 */
+	public static function rest_api_init() {
+		// Register a rest_prepare_{$post_type} filter for each one of the post types supported
+		// by the AMP plugin.
+		foreach ( AMP_Post_Type_Support::get_builtin_supported_post_types() as $post_type ) {
```

Comment:
This needs to use `AMP_Post_Type_Support::get_eligible_post_types()` instead.

Reply:
Fixed.